libs/features/mas/web-components/src/variants/ccd-slice.css.js

```diff
@@ -0,0 +1,20 @@
+import { TABLET_UP, DESKTOP_UP, LARGE_DESKTOP } from "../media.js";
```

Comment:
Looks like you have unused constants here

Reply:
having an image with the right dimensions is preferable IMO.

Reply:
removed. I have also added a 1px image to be used for test files.